File: fibs/Fibs.java
public class Fibs {
    public static void main(String[] args) {
        for (int i = 1; i < 101; i++) {
            boolean mod2 = (i % 2 == 0);
            boolean mod5 = (i % 5 == 0);
            boolean mod9 = (i % 9 == 0);
            if (mod2)
                System.out.print("Fibs");
            if (mod5)
                System.out.print("Jubs");
            if (mod9)
                System.out.print("Magoo");
            if (!mod2 && !mod5 && !mod9)
                System.out.print(i);
            System.out.println();
        }
    }
}

File: fibs/fibs.py
import sys

for i in range(1, 101):
    mod2 = (i % 2 == 0)
    mod5 = (i % 5 == 0)
    mod9 = (i % 9 == 0)
    if mod2:
        sys.stdout.write("Fibs")
    if mod5:
        sys.stdout.write("Jubs")
    if mod9:
        sys.stdout.write("Magoo")
    if not mod2 and not mod5 and not mod9:
        sys.stdout.write(str(i))
    print("")
